src/components/reusables/NewDelegationAccordion/NewDelegationAccordion.tsx

```diff
@@ -13,7 +13,7 @@ import * as React from 'react';
 import { ReactComponent as MinusCircle } from '@/assets/MinusCircle.svg';
 import { ReactComponent as AddCircle } from '@/assets/AddCircle.svg';
 
-import { Line } from '../Line/Line';
+import { Line } from '../Line';
 
 import classes from './NewDelegationAccordion.module.css';
```

Comment:
For some reason accordion looks like this for me with the plus/minus icon not on the end of the accordion. 
![image](https://user-images.githubusercontent.com/31471142/217813974-73598446-8592-460c-82e9-bed9ef7bec22.png)